managed_config/index.bs

```diff
@@ -0,0 +1,116 @@
+<pre class='metadata'>
+Title: Managed Configuration API
+Shortname: managed-configuration
+Level: 1
+Editor: Anatoliy Potapchuk https://www.w3.org/users/130190, Google LLC https://www.google.com, apotapchuk@google.com
+Status: w3c/CG-DRAFT
+Issue Tracking: Github https://github.com/WICG/WebApiDevice/issues
+URL: https://wicg.github.io/WebApiDevice/managed_config
+Abstract: This document defines a web platform API that enables developers to access administrator-set configuration on managed devices. Such API invaluable for such use cases as digital signage and device-centric personalization.
+Repository: https://github.com/WICG/WebApiDevice
+Group: wicg
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:interface; text:Navigator
+spec:url; type:dfn; text:origin
+spec:html; type:dfn; for:/; text:global object
+spec:webidl; type:dfn; text:resolve
+spec:webidl; type:dfn; text:record
+</pre>
+
+<pre class="anchors">
+
+</pre>
+
+# Model # {#model}
+
+## Managed devices ## {#managed-devices}
+
+The API is presumed to be used on devices which are not fully controlled by the end user but rather by an external entity, the device administrator. Device administrators are given the power to fully control the managed device.
+
+## Managed configuration ## {#managed-configuration}
+
+For the purposes of this specification the device administrator can control a list of managed web applications. Each entry in this list can have a set JSON configuration, which is accessible by the documents hosted at the origin of the web application.
+
+Note that this does not apply to anonymous contexts, where the observed state is indistinguishable from a non-managed one.
+
+## <dfn dfn>Data model</dfn> ## {#data-model-section}
+
+We can assume without loss of generality that managed configuration is stored in a two-level key-value store.
+
+[=Data model=] is declared as a [=map=] [=map=] where each [=key=] is a configured [=origin=] and the value is a <xmp>record<DOMString, object></xmp> with items corresponding to per-origin configuration keys and their values.
```

Comment:
Use &lt;code> instead of &lt;xmp>.

Reply:
I need xmp to ignore the braces "<>" from the template arguments

Reply:
Use "&lt;code>[=record=]&amp;lt;DOMString, object>&lt;/code>".